### DISCUSSION TITLE
restort

### DISCUSSION THREAD

**Discussion by riadiat** [community/community#116424] [New to GitHub]
Hello, the programmer deleted the Front and View js files yesterday and we want to recover the files urgently

**Comment by ghostinhershell** [comment]
Hi @riadiat, 

Thanks so much for reaching out. These resources from our Docs may be helpful for you to reference: 
- [Restoring a deleted repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository)
- [Deleting and restoring branches in a pull request](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/deleting-and-restoring-branches-in-a-pull-request)
- [GitHub Account Recovery Policy](https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy)

Thanks!
